### PR TITLE
ARM64: LegalizeDirectBranch

### PR DESCRIPTION
**Pull request by sigatrev**
update for arm64

### CODE REVIEW

File: lib/Backend/arm64/ARM64Encoder.h [hunk 1]
```diff
@@ -9,12 +9,18 @@
 
 // ToDo (SaAgarwa) - Copied from old ARMEncode.h file to make debug build
```
Comment (agarwal-sandeep) [inline lib/Backend/arm64/ARM64Encoder.h]: nit: remove

**Comment by sigatrev** [comment]
@dotnet-bot test Windows ci_slow_x64_debug

**Comment by sigatrev** [comment]
@dotnet-bot test Tab Check

**Comment by sigatrev** [comment]
@dotnet-bot test EOL Check

**Pull request merged**